src/agent/protocols/protos/health.proto

```diff
@@ -11,7 +11,7 @@ option go_package = "github.com/kata-containers/kata-containers/src/runtime/virt
 
```

Comment:
copyright update?

Reply:
Do we have to declare copyright with a year?

Can something like
```Copyright (c) Ant Group / Intel Corporation```
declare forever copyright on a file and thus we do not have to update the `year` filed every year?

/cc @jodh-intel in case you have more insights.

Reply:
I think we can't remove the year, because the copyright notice require the year of first publication of the work.

Reply:
If it only requires the year of first publication, we do not need to update the `year` field later on, is it correct?

Reply:
Yes, I meant we should update the company name

Reply:
But many people like to use 2019-2020 pattern, and it is ok though the copyright law does not mention it. The new pattern is quite popular in recent years.

Reply:
hmm, I see your point. It is `Ant Group` now...